src/lib.rs

```diff
@@ -148,12 +166,18 @@ impl<const T: usize> DataMut for List<T> {
     fn data_mut(&mut self)->&mut[Elem] { &mut self.data }
 }
 
-trait Use { fn use_idx(&mut self, i: u8);}
-impl Use for Pair {fn use_idx(&mut self, _:u8){}}
+trait Use {
+  fn use_idx(&mut self, i: u8);
+}
+
+impl Use for Pair {
+  fn use_idx(&mut self, _:u8) {}
+}
+
```

Comment:
I imagine you can already guess my objection to respacing things from 2 to 7 lines, or above from 30 to 50 lines. Even under "a line should do only one thing", defining a single overloadable function is one thing, as is implementing it for `Pair` as a nop